pipelines/common/src/test/java/com/google/fhir/analytics/AvroConversionUtilTest.java

```diff
@@ -131,6 +131,7 @@ public void generateRecords_BundleOfObservations() throws ProfileException {
     assertThat(recordList.size(), equalTo(6));
   }
 
+  // SuppressWarnings added to ignore testcase NullAway warnings.
```

Comment:
This is basically saying the same thing as the actual line below, I mean it does not add much. As I said in the last review, it is okay if you want to leave some of these `@SuppressWarnings` without a comment, but if you add a comment it should be something more. Please drop all the generic comments.

Reply:
Noted. Reverting these.